FSW/main.cpp

```diff
@@ -14,10 +14,40 @@
 // Other libraries
 
 // NS2 headers
+#include "src/headers/config.hpp"
+#include "src/headers/comUtil.hpp"
+#include "src/headers/eventUtil.hpp"
+#include "src/headers/timingClass.hpp"
+#include "src/headers/timing.hpp"
 #include "src/headers/memUtil.hpp"
 
 /* - - - - - - Functions - - - - - - */
 
+/* - - - - - - init - - - - - - *
+ * Usage:
+ *  initializes NanoSAM II FSW
+ * 
+ * Inputs:
+ *  none
+ *  
+ * Outputs:
+ *  initStatus - flag (true if initialization was successful)
+ */
+bool init(){
+    Serial.print("Initializing NanoSAM II FSW... ");
+
+    scienceMode.setPointingAtSun(true);
+    scienceMode.setMode(SAFE_MODE); // this is done in the class constructor so we may not need it
```

Comment:
Why do we start in safe mode and not the nominal mode?

Reply:
should be `STANDBY_MODE`, updated. Good catch